frontend/src/metabase/querying/filters/components/FilterPicker/FilterSubmitButton/FilterSubmitButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { t } from "ttag";
+
+import { Button, Divider, Icon, Tooltip } from "metabase/ui";
+
+type FilterSubmitButtonProps = {
+  isNew?: boolean;
+  isDisabled?: boolean;
+  withAddButton?: boolean;
+  onAddButtonClick?: () => void;
+};
+
+export function FilterSubmitButton({
+  isNew,
+  isDisabled,
+  withAddButton,
+  onAddButtonClick,
+}: FilterSubmitButtonProps) {
+  if (!withAddButton) {
+    return (
+      <Button type="submit" variant="filled" disabled={isDisabled}>
+        {isNew ? t`Add filter` : t`Update filter`}
+      </Button>
+    );
+  }
+
+  const addButtonLabel = t`Add another filter`;
+  return (
+    <Button.Group>
+      <Button type="submit" variant="filled" disabled={isDisabled}>
+        {t`Apply filter`}
+      </Button>
+      <Divider orientation="vertical" />
+      <Tooltip label={addButtonLabel}>
+        <Button
+          type="button"
+          variant="filled"
+          disabled={isDisabled}
+          leftSection={<Icon name="add" />}
+          aria-label={addButtonLabel}
+          onClick={onAddButtonClick}
```

Comment:
When this button is clicked the search input in `MultiStageFilterPicker` should be autofocused, but (most of the time) the focus is immediately being lost. 
It seems that `data-testid="native-query-preview-sidebar"` is rendered in chill mode and code mirror steals the focus.

https://github.com/user-attachments/assets/9f5c675e-6b10-44cd-80b2-0acf75c18b66